frontend/src/packages/auth/libs/types/types.ts

```diff
@@ -0,0 +1 @@
+export { type UserAuthResponseDto } from 'shared/build/index.js';
```

Comment:
As it is now reexported through `package/users`
```suggestion

```

Reply:
It was added after the merge(